### ISSUE TITLE
Test fails for some exotic architectures within Debian Sid environment

### ISSUE THREAD

**Issue by jgmbenoit** [Normaliz/Normaliz#168]
More details can be found at the current [Debian Package Auto-Building page](https://buildd.debian.org/status/package.php?p=normaliz). Please let me know how I can help.

**Comment by w-bruns** [comment]
1. Dependency installability problem: I don't see what I can do.I will forward the llink to Bill Hart.
2. Several failures of terst-q/bugHpoly: my first attempt to exclude certain lines from diff failed. With the help of Sebastian Gutsche  this has been corrected. But without seeing the diff I cannot analyze the problem.
3. Failure of Qtest/test-s/cut_poly_7 with segmentation fault on  mips64el: no idea what it could be. 
4. Compiler error on  powerpcspe: No idea. I cannot access the file suggested.

I have uploaded a new tar ball for the source containing the missing files (Docker/Dockerfile, doc/jNormaliz.jpg) and with a corrected test/Makefile.classic taking hopefully care of the diff issue.
